core/LogUtils.hpp

```diff
@@ -0,0 +1,21 @@
+// <LogUtils.hpp> -*- C++ -*-
```

Comment:
This file is already in Sparta:  sparta/utils/LogUtils.hpp

Reply:
Ok